lib/perm/own-badge.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function ownBadge (args, callback) {
+    var seneca = this;
+    var requestUser = args.user.id;
+    var badgeUser = args.params.badgeData.userId;
+    var validity = false;
```

Comment:
If you ensure me that equality cannot return anything than a boolean in Js, i sign :)